src/node/in_memory.rs

```diff
@@ -1171,34 +1171,39 @@ impl<S: ForkSource + std::fmt::Debug + Clone> InMemoryNode<S> {
             .take()
             .unwrap_or_default();
 
-        match &tx_result.result {
-            ExecutionResult::Success { output } => {
-                tracing::info!("Call: {}", "SUCCESS".green());
-                let output_bytes = zksync_types::web3::Bytes::from(output.clone());
-                tracing::info!("Output: {}", serde_json::to_string(&output_bytes).unwrap());
-            }
-            ExecutionResult::Revert { output } => {
-                tracing::info!("Call: {}: {}", "FAILED".red(), output);
-            }
-            ExecutionResult::Halt { reason } => {
-                tracing::info!("Call: {} {}", "HALTED".red(), reason)
-            }
-        };
+        if !inner.config.show_only_contract_logs {
+            match &tx_result.result {
+                ExecutionResult::Success { output } => {
+                    tracing::info!("Call: {}", "SUCCESS".green());
+                    let output_bytes = zksync_types::web3::Bytes::from(output.clone());
+                    tracing::info!("Output: {}", serde_json::to_string(&output_bytes).unwrap());
+                }
+                ExecutionResult::Revert { output } => {
+                    tracing::info!("Call: {}: {}", "FAILED".red(), output);
+                }
+                ExecutionResult::Halt { reason } => {
+                    tracing::info!("Call: {} {}", "HALTED".red(), reason)
+                }
+            };
+            // Don't print the header if only contract calls should be shown
+            tracing::info!("=== Console Logs: ");
+        }
```

Comment:
I dont think we should disable informing on reverts, halts, successes via this `show_only_contract_logs`. This flag should only toggle console logs, no?

Reply:
See point made above

Reply:
Removed show_only_contract_logs and added individual flags to control every section of logs